protobuf/google/rpc/README.md

```diff
@@ -0,0 +1,17 @@
+# Google RPC
+
+This package contains type definitions for general RPC systems. While
+[gRPC](https://github.com/grpc) is using these defintions, they
+are not designed specifically to support gRPC.
+
+## `containerd`
+
+This is moved from the [googleapis
+project](https://github.com/googleapis/googleapis/tree/master/google/rpc) to
```

Comment:
Please put git commit hash here?

Also, I think this directory should have LICENSE file.

Reply:
LICENSEs are at the top of each file.

Reply:
@caniszczyk Is that enough to meet requirements?